.circleci/config.yml

```
@@ -199,7 +221,7 @@ workflows:
                 - /^release.*/
       - UI Tests:
           name: UI Tests (iPad Air 4th generation)
```

To test this PR, I changed the iPhone version from iPhone 11 to iPhone 12. When I went to unblock the UI tests, I was surprised for a second they mentioned "iPhone 11".

I'd suggest removing the device name from the job name.


```suggestion
          name: UI Tests iPad
```